.github/workflows/main.yml

```
@@ -21,25 +21,27 @@
 
       - name: Build Docker image
-        run: docker build -t testnew:latest .
+        run: docker compose build
 
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@0.28.0
         env:
           TRIVY_DISABLE_VEX_NOTICE: "true"
         with:
-            image-ref: testnew:latest
+            image-ref: devdox:latest
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)